Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/TextBoxTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Aquality.WinAppDriver.Applications;
+using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Tests.Applications.Locators;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Aquality.WinAppDriver.Tests.Elements
+{
+    public class TextBoxTests : TestWithApplication
+    {
+        private const string ExpectedValue = "1";
+        private IElementFactory Factory => ApplicationManager.GetRequiredService<IElementFactory>();
+
+        [Test]
+        public void Should_EnterValues()
+        {
+            var textBox = GetTextBox(Factory, CalculatorWindow.RightArgumentTextBox, "Right Argument");
```

Comment:
we do always create this textbox with the same parameters.
let's have a private field ITextBox RightArgumentTextBox for it, seem that we don't need a general GetTextBox method since we don't have any other textboxes

Reply:
done